calico/README.md

```diff
@@ -0,0 +1,155 @@
+# Agent Check: Calico
+
+## Overview
+
+This check monitors [Calico][1] through the Datadog Agent.
+
+The Calico check sends metrics concerning network and security in a Kubernetes cluster set up with Calico.
+
+## Setup
+
+### Installation
+
+The Calico check is included in the [Datadog Agent][2] package, to start gathering metrics you first need to:
+
+#### Installation with a Kubernetes cluster-based Agent
+
+Using annotations:
+
+1. Set up Calico on your cluster if you have not already.
+
+2. Enable Prometheus metrics using the instructions in [Monitor Calico component metrics][9].
+   Once enabled, you should have a `felix-metrics-svc` service running in your cluster, as well as a `prometheus-pod`.
+
+3. To use Autodiscovery, modify `prometheus-pod`. Add the following snippet to your Prometheus YAML configuration file:
+
+```
+metadata:
+  [...]
+  annotations:
+   ad.datadoghq.com/prometheus-pod.check_names: |
+   ["openmetrics"]
+   ad.datadoghq.com/prometheus-pod.init_configs: |
+   [{}]
+   ad.datadoghq.com/prometheus-pod.instances: |
+     [
+        {
+           "prometheus_url": "http://<FELIX-SERVICE-IP>:<FELIX-SERVICE-PORT>/metrics",
+           "namespace": "calico",
+           "metrics": ["*"]
+        }
+     ]
+  spec:
+    [....]
+```
```

Comment:
Minor nit but if you want this code block to follow the same tabbed formatting as the instructions, you can move this code block by three spaces so that the first backtick is the same place below "To use..."